### ISSUE TITLE
Links don't work

### ISSUE THREAD

**Issue by vinayvenu** [iamacup/react-native-markdown-display#40]
When there is no customCallback, links do not automatically work. 

#### Sample test
```
<Markdown >{`[This is a link!](https://github.com/iamacup/react-native-markdown-display/)`}</Markdown>
```

#### Workaround
Add LInking.openURL to the callback. 

```
<Markdown **onLinkPress={(url) => Linking.openURL(url)}** >{`[This is a link!](https://github.com/iamacup/react-native-markdown-display/)`}</MTMarkdown>
```

We look for existence of customCallBack in [openURL.js](https://github.com/iamacup/react-native-markdown-display/blob/master/src/lib/util/openUrl.js#L4), but the [component definition](https://github.com/iamacup/react-native-markdown-display/blob/master/src/index.js#L112) already provides a default. This makes the else condition unreachable.

**Comment by vinayvenu** [comment]
PR https://github.com/iamacup/react-native-markdown-display/pull/41

**Comment by iamacup** [comment]
released as 4.0.13 on npm
